base/array.jl

```diff
@@ -1251,10 +1251,11 @@ julia> findnext(A,3)
 function findnext(A, start::Integer)
     for i = start:length(A)
         if A[i] != 0
-            return i
```

Comment:
Afterthought: it's also a bit more 'functional' because it's easier to reason about whether the resulting variable `v` is mutable or not.